otter/test/test_controller.py

```diff
@@ -1104,6 +1105,30 @@ def test_real_convergence(self):
         result = controller.converge(log, 'txn-id', group_config, self.group,
                                      state, 'launch', policy,
                                      config_value=config_data.get)
+        self.assertIsInstance(result, defer.Deferred)
```

Comment:
IsInstance is probably overkill here, I think just giving it to successResultOf is good enough